server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleIndex.java

```diff
@@ -379,6 +379,8 @@ private static void addDefaultFacets(RuleQuery query, SearchOptions options, Map
     }
   }
 
+  // TODO add type facets
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)